### PR TITLE
[receiver/kubeletstats] Start name change for cpu.utilization

### PR DESCRIPTION
**Pull request by TylerHelmuth**
**Description:** 
Starts the name change processor for `*.cpu.utilization` metrics. 

**Link to tracking Issue:** 
Related to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/24905
Related to https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/27885

**Testing:**

### CODE REVIEW

**Comment by TylerHelmuth** [comment]
@povilasv good questions, can you ask that question in the issue

**Review by povilasv — APPROVED** [review]

**Review by dmitryax — COMMENTED** [review]
It's unclear how the new `usage` metrics are different from `time` metrics. Can you please clarify that they are average aggregations over time?

**Comment by TylerHelmuth** [comment]
@dmitryax I'm only as knowledgable as the [comments on the struct](https://pkg.go.dev/k8s.io/kubelet/pkg/apis/stats/v1alpha1#CPUStats):

```go
// CPUStats contains data about CPU usage.
type CPUStats struct {
	// The time at which these stats were updated.
	Time metav1.Time `json:"time"`
	// Total CPU usage (sum of all cores) averaged over the sample window.
	// The "core" unit can be interpreted as CPU core-nanoseconds per second.
	// +optional
	UsageNanoCores *uint64 `json:"usageNanoCores,omitempty"`
	// Cumulative CPU usage (sum of all cores) since object creation.
	// +optional
	UsageCoreNanoSeconds *uint64 `json:"usageCoreNanoSeconds,omitempty"`
}
```

We use `Time` to set the `*.cpu.time` and we use `UsageNanoCores` to set `*.cpu.utilization` (and now `*.cpu.usage`). @jinja2 do you know?

**Comment by dmitryax** [comment]
I was just asking to improve the description of the "usage" metrics to make it easier to differentiate it from the "time" metrics.

Usage can be described as "Total CPU usage (sum of all cores per second) averaged over the sample window". 

Time can be "Total cumulative CPU time (sum of all cores) spent by the container/pod/node since its creation".

cc @povilasv @jinja2 let me know if it's clear enough

File: receiver/kubeletstatsreceiver/metadata.yaml [hunk 1]
```diff
@@ -81,9 +81,18 @@ attributes:
     enum: [receive, transmit]
 
 metrics:
+  k8s.node.cpu.usage:
+    enabled: false
+    description: "Node CPU usage"
+    unit: s
```
Comment (dmitryax) [inline receiver/kubeletstatsreceiver/metadata.yaml:87]: Not sure unit is `s`. Maybe `{cpu}`?

**Review by dmitryax — APPROVED** [review]
LGTM. @povilasv PTAL

**Review by povilasv — APPROVED** [review]
LGTM

**Pull request merged**